.packit.yaml

```
@@ -1,47 +1,61 @@
 specfile_path: dist/beakerlib.spec
-#synced_files:
-#    - beakerlib.spec
+
+files_to_sync:
+  - src: dist/*
+    dest: .
+  - .packit.yaml
 
 upstream_package_name: beakerlib
 downstream_package_name: beakerlib
 
-
-#actions:
-#  create-archive:
-#  - make tarball
-#  get-current-version:
-#  - make version
-
 jobs:
 - job: copr_build
   trigger: pull_request
-  metadata:
-    targets:
-    - fedora-all
-    - centos-stream-8
-    - centos-stream-9
-    - epel-6
-    - epel-all
+  targets:
+  - fedora-all
+  - centos-stream-8
+  - centos-stream-9
+  - epel-6
+  - epel-all
 
 - job: copr_build
   trigger: commit
-  metadata:
-    branch: master
-    targets:
-    - fedora-all-x86_64
-    - fedora-all-aarch64
-    - centos-stream-8-x86_64
-    - centos-stream-8-aarch64
-    - centos-stream-9-x86_64
-    - centos-stream-9-aarch64
-    - epel-6-x86_64
-    - epel-6-aarch64
-    - epel-all-x86_64
-    - epel-all-aarch64
+  branch: master
+  targets:
+  - fedora-all-x86_64
+  - fedora-all-aarch64
+  - centos-stream-8-x86_64
+  - centos-stream-8-aarch64
+  - centos-stream-9-x86_64
+  - centos-stream-9-aarch64
+  - epel-6-x86_64
+  - epel-6-aarch64
+  - epel-all-x86_64
+  - epel-all-aarch64
 
 - job: tests
   trigger: pull_request
-  metadata:
-    targets:
+  targets:
+  - fedora-all
+  - epel-8
+
+- job: propose_downstream
+  trigger: release
+  dist_git_branches:
+    - fedora-rawhide
+    - fedora-branched
```

```suggestion
    - fedora-all
```

This should be the same.